components/form/index.en-US.md

```diff
@@ -275,3 +275,55 @@ validator(rule, value, callback) => {
   }
 }
 ```
+
+### How you can use `wrappedComponentRef` to get form instance in functional component
+
+you have to combine `forwardRef` with `useImperativeHandle` so you can get form instance by using `wrappedComponentRef`:
```

Comment:
> You should wrap function component with `forwardRef` to pass ref:

Reply:
`useImperativeHandle` is not must have. Please help to check this.

Reply:
https://reactjs.org/docs/hooks-reference.html#useimperativehandle

Reply:
`useImperativeHandle` is aim to customize `ref` content. `wrappedComponentRef` get the form instance with Form.create. They not use customize `ref` indeed.

Reply:
well, you might be right, but in fact, if I remove `useImperativeHandle`, I cannot get the instance. you can try the codesandbox.